Makefile

```diff
@@ -2,7 +2,8 @@ ERROR=echo "\033[0;31m"
 SUCCESS=echo "\033[0;32m"
 
 CARGO_RUN_WATCH = RUST_LOG=debug cargo watch -x 'run --bin social-service -- --port 8080'
-CARGO_RUN = RUST_LOG=debug cargo run -- --port 8080
+# TODO: remove `--bin social-service`
```

Comment:
Is this still a valid TODO?

Reply:
Yes, as long as we have the `client.rs` and this block in out Cargo.toml
```
[[bin]]
name = "client-ws"
path = "src/client.rs"
```